catalogue/webapp/components/SearchPageLayout/SearchPageLayout.tsx

```diff
@@ -44,96 +44,108 @@ const SearchLayout: FunctionComponent<{ hasEventsExhibitions: boolean }> = ({
 
   const getURL = pathname => {
-    const query = { query: router.query.query };
     return convertUrlToString({
       pathname,
-      query,
+      query: { query: queryString },
     });
   };
 
   useEffect(() => {
-    const { query } = router.query;
+    const queryStringTitle = queryString ? `${queryString} | ` : '';
 
     switch (currentSearchCategory) {
       case 'overview':
         setPageLayoutMetadata({
           ...basePageMetadata,
-          description: pageDescriptions.search.overview,
-          title: `${query ? `${query} | ` : ''}Search`,
-          url: { pathname: '/search', query: { query } || {} },
+          title: `${queryStringTitle}Search`,
         });
         break;
       case 'stories':
         setPageLayoutMetadata({
           ...basePageMetadata,
           description: pageDescriptions.search.stories,
-          title: `${query ? `${query} | ` : ''}Stories search`,
-          url: { pathname: '/search/stories', query: { query } || {} },
+          title: `${queryStringTitle}Stories search`,
+          url: {
+            ...basePageMetadata.url,
+            pathname: '/search/stories',
+          },
         });
         break;
       case 'images':
         setPageLayoutMetadata({
           ...basePageMetadata,
           description: pageDescriptions.search.images,
-          title: `${query ? `${query} | ` : ''}Image search`,
-          url: { pathname: '/search/images', query: { query } || {} },
+          title: `${queryStringTitle}Image search`,
+          url: {
+            ...basePageMetadata.url,
+            pathname: '/search/images',
+          },
         });
         break;
       case 'works':
         setPageLayoutMetadata({
           ...basePageMetadata,
           description: pageDescriptions.search.works,
-          title: `${query ? `${query} | ` : ''}Catalogue search`,
-          url: { pathname: '/search/works', query: { query } || {} },
+          title: `${queryStringTitle}Catalogue search`,
+          url: {
+            ...basePageMetadata.url,
+            pathname: '/search/works',
+          },
         });
         break;
       // In development
       case 'exhibitions':
         setPageLayoutMetadata({
           ...basePageMetadata,
           description: pageDescriptions.search.exhibitions,
-          title: `${query ? `${query} | ` : ''}Exhibition Search`,
-          url: { pathname: '/search/exhibitions', query: { query } || {} },
+          title: `${queryStringTitle}Exhibition Search`,
+          url: {
+            ...basePageMetadata.url,
+            pathname: '/search/exhibitions',
+          },
         });
         break;
       case 'events':
         setPageLayoutMetadata({
           ...basePageMetadata,
           description: pageDescriptions.search.events,
-          title: `${query ? `${query} | ` : ''}Events Search`,
-          url: { pathname: '/search/events', query: { query } || {} },
+          title: `${queryStringTitle}Events Search`,
```

Comment:
nit: singular Event and lowercase search for consistency

Reply:
I just checked with Danny and we'll go plural everywhere actually